docs/how-to/setup-time-sources.md

```diff
@@ -0,0 +1,28 @@
+# How to: Setup Time Sources
+
+Chrony alone cannot provide time; it requires another time source, either a 
+reference clock or another NTP server, to supply time to Chrony itself. 
+Currently, the Chrony charm supports only using another NTP server as the 
```

Comment:
Nitpick: I would flip the order of "supports" and "only", in other words:
`Currently, the Chrony charm only supports using another NTP...`

Reply:
Updated, thanks!